st2reactor/st2reactor/timer/base.py

```diff
@@ -132,8 +134,9 @@ def _emit_trigger_instance(self, trigger):
             'executed_at': str(utc_now),
             'schedule': trigger['parameters'].get('time')
         }
+
         trace_context = TraceContext(trace_tag='%s-%s' % (self._get_trigger_type_name(trigger),
-                                                          trigger.name))
+                                                          trigger.get('name', uuid.uuid4().hex)))
```

Comment:
Is it possible to have a trigger without name at this point?

We randomly generate one if one is not provided so it seems like this shouldn't be possible.